lib/ansible/modules/windows/win_domain.ps1

```diff
@@ -17,6 +17,8 @@ Function Ensure-Prereqs {
         # NOTE: AD-Domain-Services includes: RSAT-AD-AdminCenter, RSAT-AD-Powershell and RSAT-ADDS-Tools
         $awf = Add-WindowsFeature AD-Domain-Services -WhatIf:$check_mode
         # FUTURE: Check if reboot necessary
+        $result.reboot_required = $awf.RestartNeeded
```

Comment:
The changes in this module should be in a separate PR as they modify how the module behaves.

Reply:
Honestly, I just added it on the fly since it seemed the most logical thing for that section, I don't mind removing it of course

Reply:
It's not wrong, but it belongs in another PR where it will be discussed/accepted on its own merits.

Reply:
removed